pkg/kubelet/lifecycle/predicate.go

```diff
@@ -84,8 +79,13 @@ func (w *predicateAdmitHandler) Admit(attrs *PodAdmitAttributes) PodAdmitResult
 			Message: message,
 		}
 	}
-	meta := w.metadataProducer(pod, map[string]*schedulercache.NodeInfo{node.Name: nodeInfo})
-	fit, reasons, err := predicates.GeneralPredicates(pod, meta, nodeInfo)
+
+	// Remove the requests of the extended resources that are missing in the
+	// node info. This is required to support cluster-level resources, which
+	// are extended resources unknown to nodes.
```

Comment:
There is an issue where if a pod was manually (or incorrectly) bound to a node where a node level extended resource it requires is not found, then kubelet will not fail admission. We should document this as known issue and fix it with resource APIs.

Reply:
Will do.